docs/contributing/website.md

```diff
@@ -13,6 +13,8 @@ The current documentation website build reuses some of these files, specifically
 - Entirety of `CONTRIBUTING.md`
 - Entirety of `CODE_OF_CONDUCT.md`
 - Entirety of `CHANGELOG.md`
+- Entirely of `MAINTAINERS.md`
+- Entirely of `DCO1.1.txt`
```

Comment:
Please confirm you wanted to add these here.  I thought you had found a reason they should be separate but I may have misunderstood.

Reply:
These are those files which are used from the main github project directory. The markdown files are directly included in the corresponding files in docs directory, while the LICENSE and DOC and text files and are include using docs automation script to convert them to markdown format.